src/ConfigEditor.tsx

```diff
@@ -35,6 +39,16 @@ export class ConfigEditor extends PureComponent<Props, State> {
               />
             </InlineField>
           </div>
+          <div className="gf-form-inline">
+            <FormField
```

Comment:
I don't know much about this UI part, so excuse my naive question - but why use FormField instead of InlineField?

Reply:
I was following some other guides about the form development, but it seems like the recommendation is to use InlineField and then Input so I switched to those